flexmeasures/ui/crud/users.py

```diff
@@ -74,20 +74,15 @@ def process_internal_api_response(
     return user_data
 
 
-def get_users_by_account(
-    account_id: int | str, include_inactive: bool = False
-) -> list[User]:
+def get_users_by_account(account_id: int | str, include_inactive: bool = False) -> list:
```

Comment:
This became less explicit. Returns a list of what? For example, in case of dicts, let's use ` -> list[dict]`.